specification/schemas/MinimalOwner.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
There is no owner type, so there is no 'minimal owner', there is only a relationship with the name `owner`. So this should be removed.